CONTRIBUTING.md

```diff
@@ -94,7 +94,10 @@ When a commit is made on the `master` branch, documentation is automatically bui
 
 ## JavaScript dependencies
 
-For the time being, we are committing JavaScript dependencies to the repository as that makes installation much easier.
+For the time being, we are committing JavaScript dependencies to the repository as that makes installation much easier. However there is an unbundled dependency on PhantomJS that must be installed separately. If you have `npm` installed, you can install PhantomJS by running:
+    
+    npm install phantomjs
```

Comment:
Not everyone will necessarily have `npm` installed, so I think this should also include instructions for installing with `apt-get` and homebrew.